apps/remix-ide/src/app/compiler/compiler-utils.js

```diff
@@ -25,8 +25,10 @@ export function canUseWorker (selectedVersion) {
   const version = semver.coerce(selectedVersion)
   const isNightly = selectedVersion.includes('nightly')
   return browserSupportWorker() && (
+    // All compiler versions (including nightlies) after 0.6.3 are wasm compiled
```

Comment:
```suggestion
    // All compiler versions (including nightlies) starting with 0.6.2 are wasm compiled
```

Not sure if all 0.6.2 nightlies are wasm. I'll have to check that.

Reply:
those are also wasm Daniel said

Reply:
ok. Just checked to confirm and I see that `soljson-v0.6.2+commit.bacdbe57.js` (i.e. the first nightly of the 0.6.2 series) seems to be the the first one compiled to wasm. `soljson-v0.6.1-nightly.2020.1.7+commit.8385256b.js` (last in 0.6.1) was asm.js.